spec/dummy/config/application.rb

```diff
@@ -24,6 +24,72 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
     config.i18n.default_locale = :en
     config.i18n.fallbacks = [I18n.default_locale]
+    config.i18n.available_locales = %i[
```

Comment:
Does this mean that this list will need to be manually updated when a new translation file is added?

Reply:
Makes sense - and I can just see this being a thing that gets left out if another language is added. Is there any way that this list and the list of translation files could be compared to make sure that no language is missing? Or failing that, could a quick "So you want to add another language" guide be added to the [component gem's docs][gd]?

[gd]: https://github.com/alphagov/govuk_publishing_components/tree/master/docs